recipes/nflows/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "nflows" %}
+{% set version = "0.14" %}
+{% set python_min = "3.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/nflows-{{ version }}.tar.gz
+  sha256: 6299844a62f9999fcdf2d95cb2d01c091a50136bd17826e303aba646b2d11b55
+
+build:
+  noarch: python
```

Comment:
Windows builds will fail here as expected, given that PyTorch doesn't support Windows. The rest is `noarch` though so can keep it as `noarch: python`.